ReplayMessenger.device.lib.nut

```diff
@@ -93,6 +109,41 @@ class ReplayMessenger {
         return _mm.send(messageName, data, null, _retryInterval, metadata);
     }
 
+    function onReplayBegin(loggerName, callback = null) {
+      if (callback == null && typeof loggerName == "string") { //de-register callback
+        if (loggerName in _onReplayBegin) {
+          delete _onReplayBegin[loggerName]
+        }
+      } else if (callback == null) { //assume callback is provided for "default" logger
```

Comment:
No it still doesn't make sense to me. I mean the code doesn't do what you expect it to do I guess. It doesn't set a handler. What it does it changes the local variables which are never used then.